### ISSUE TITLE
accessibility focus lose from the ListView.builder() on a specific accessibility device 

### ISSUE THREAD

**Issue by yama3nomori** [flutter/flutter#68022]
Following codes are the same action on most of devices.
Of course, it can swipe and scroll items of the list view using TalkBack.

But, following device is not the same action.

HIMS Inc. | BrailleSense Polaris  
https://hims-inc.com/product/braillesense-polaris/

The code I is no problem.
But the codeII is a little problem.

User can scroll a list view by swipe action.
But accessibility focus lose from the list view.
If there is a actionbutton or bottom navigation bar, accessibility focus move them.
After that, focus move top position.
It's incorrectly.

I'm thinking ListView.builder doesn't have actions of SinglechildScrollView.
Please fix it.
BecauseListView.builder is very conveniently.

// code I:
```
SingleChildScrollView(
child: ListBody(
  children: [
    Text('item 0'),
    Text('item 1'),
    Text('item 2'),
    // There're many text widgets.
    // ...
    Text('item 99'),
  ],
),
);


// code II:
ListView.builder(
itemCount: 100,
itemBuilder: (context, index){
  return Text('item $num');
},
);
```

**Comment by TahaTesser** [comment]
Hi @yama3nomori 
Never seen this device, can you please provide details about the issue and the device it'shappeing on
Can you please provide your `flutter doctor -v` your `flutter run --verbose` and a complete reproducible minimal code sample
Thank you

**Comment by yama3nomori** [comment]
Hello, @TahaTesser

Thanks for your comment.
Additional infomation is following:


Device's detail is here:
https://hims-inc.com/hims-trade-in-specials/



'flutter doctor -v' is following:

[√] Flutter (Channel beta, 1.18.0-11.1.pre, on Microsoft Windows [Version 10.0.19041.508], locale ja-JP)
    • Flutter version 1.18.0-11.1.pre at C:\Android\flutter
    • Framework revision 2738a1148b (5 months ago), 2020-05-13 15:24:36 -0700
    • Engine revision ef9215ceb2
    • Dart version 2.9.0 (build 2.9.0-8.2.beta)

[!] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at C:\Android\sdk
    • Platform android-29, build-tools 29.0.2
    • ANDROID_HOME = C:\Android\sdk
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)
    X Android license status unknown.
      Try re-installing or updating your Android SDK Manager.
      See https://developer.android.com/studio/#downloads or visit visit https://flutter.dev/docs/get-started/install/windows#android-setup for detailed instructions.

[!] Android Studio (version 3.5)
    • Android Studio at C:\Program Files\Android\Android Studio
    X Flutter plugin not installed; this adds Flutter specific functionality.
    X Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_202-release-1483-b03)

[√] VS Code (version 1.50.0)
    • VS Code at C:\Users\nyama\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 2.21.0

[!] Connected device
    ! No devices available

! Doctor found issues in 2 categories.



// code I':
Widget build(BuildContext context) {
List <Widget> list = List <Widget>();
for(int i = 0; i < 100; i++) {
  list.add(Text('item $i'));
}
return Scaffold(
    appBar: AppBar(
        title: Text('sample I'),
  body: Container(
    height: 300,
    width: 300,
    child: SingleChildScrollView(
child: ListBody(
  children: list,
),
  ),
  ),
  bottomNavigationBar: BottomAppBar(
    child: FloatingActionButton(
      tooltip: 'close',
      onPressed: () => Navigator.pop(context),
  ),
  ),
  );
}


// code II':
Widget build(BuildContext context) {
return Scaffold(
  appBar: AppBar(
    title: Text('sample II'),
  ),
  body: Container(
    width: 300,
    height: 300,
    child: ListView.builder(
itemCount: 100,
itemBuilder: (context, index){
  return Text('item $num');
},
  ),
  bottomNavigationBar: BottomAppBar(
    child: FloatingActionButton(
      tooltip: 'close',
      onPressed: () => Navigator.pop(context),
  ),
  ),
  );
}

**Comment by TahaTesser** [comment]
Hi @yama3nomori 
Thanks for providing the details

I can see focus moves away from listview in both code samples, Is this same the same behavior you're experiencing, please check the videos below and let me know. 

### Code 1 on Anddroid 10 
[code_1_test.zip](https://github.com/flutter/flutter/files/5376903/code_1_test.zip)
<details> 
<summary>complete code sample</summary> 

```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Material App',
      theme: ThemeData.dark(),
      home: Home(),
    );
  }
}

class Home extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    List<Widget> list = List<Widget>();
    for (int i = 0; i < 100; i++) {
      list.add(Text('item $i'));
    }
    return Scaffold(
      appBar: AppBar(
        title: Text('sample I'),
      ),
      body: Container(
        height: 300,
        width: 300,
        child: SingleChildScrollView(
          child: ListBody(
            children: list,
          ),
        ),
      ),
      bottomNavigationBar: BottomAppBar(
        child: FloatingActionButton(
          tooltip: 'close',
          onPressed: () => Navigator.pop(context),
        ),
      ),
    );
  }
}

```

</details> 


### Code 2 on Anddroid 10 
[code_2_test.zip](https://github.com/flutter/flutter/files/5376900/code_2_test.zip)

<details> 
<summary>complete code sample</summary> 

```dart
import 'package:flutter/material.dart';

void main() => runApp(MyApp());

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Material App',
      theme: ThemeData.dark(),
      home: Home(),
    );
  }
}

class Home extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      appBar: AppBar(
        title: Text('sample II'),
      ),
      body: Container(
        width: 300,
        height: 300,
        child: ListView.builder(
          itemCount: 100,
          itemBuilder: (context, index) {
            return Text('item $num');
          },
        ),
      ),
      bottomNavigationBar: BottomAppBar(
        child: FloatingActionButton(
          tooltip: 'close',
          onPressed: () => Navigator.pop(context),
        ),
      ),
    );
  }
}

```

</details> 


<details> 
<summary>flutter doctor -v</summary> 

```bash
[✓] Flutter (Channel stable, 1.22.1, on Mac OS X 10.15.7 19H2, locale en-GB)
    • Flutter version 1.22.1 at /Users/tahatesser/Code/flutter_stable
    • Framework revision f30b7f4db9 (6 days ago), 2020-10-08 10:06:30 -0700
    • Engine revision 75bef9f6c8
    • Dart version 2.10.1

 
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.2)
    • Android SDK at /Users/tahatesser/Code/sdk
    • Platform android-30, build-tools 30.0.2
    • ANDROID_HOME = /Users/tahatesser/Code/sdk
    • Java binary at: /Applications/Android
      Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.0.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.0.1, Build version 12A7300
    • CocoaPods version 1.10.0.rc.1

[!] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build
      1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.50.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.15.0

[✓] Connected device (1 available)
    • RMX2001 (mobile) • EUYTFEUSQSRGDA6D • android-arm64 • Android 10 (API 29)

! Doctor found issues in 1 category.
```

</details>

**Comment by yama3nomori** [comment]
Hi, TahaTesser

Thanks for videos.
Sorry, I cannot seethat because I'm blind.

My suggestion is minor cases.

 1. TalkBack is enabled.
 2. Device is BrailleSense.

**Comment by darshankawar** [comment]
@yama3nomori 
Revisiting the issue, please confirm if you are still seeing the reported behavior on the said device with latest stable version of the framework. Due to lack of specific device, we can't replicate at our end.
